kytos/core/logs.py

```diff
@@ -27,14 +27,14 @@ def add_stream_handler(cls, stream):
         Args:
             stream: Object that supports ``write()`` and ``flush()``.
         """
-        handler = logging.StreamHandler(stream)
+        handler = StreamHandler(stream)
         cls._add_handler(handler)
         return handler
 
     @staticmethod
     def _add_handler(handler):
-        handler.setFormatter(logging.Formatter(FORMAT))
-        logging.getLogger().addHandler(handler)
+        handler.setFormatter(Formatter(FORMAT))
```

Comment:
Can't you retrieve the format from one of those defined on the `logging.ini` file?